common/src/main/kotlin/streams/config/StreamsConfig.kt

```diff
@@ -124,4 +129,9 @@ class StreamsConfig(logService: LogService, private val dbms: DatabaseManagement
     fun hasProceduresGloballyEnabled() = this.config.getOrDefault(PROCEDURES_ENABLED, PROCEDURES_ENABLED_VALUE).toString().toBoolean()
 
     fun hasProceduresEnabled(dbName: String) = this.config.getOrDefault("${PROCEDURES_ENABLED}.$dbName", hasProceduresGloballyEnabled()).toString().toBoolean()
+
+    fun isSinkGloballyEnabled() = this.config.getOrDefault(SINK_ENABLED, SINK_ENABLED_VALUE).toString().toBoolean()
+
+    fun isSinkEnabled(dbName: String) = this.config.getOrDefault("${SINK_ENABLED}.to.$dbName", isSinkGloballyEnabled()).toString().toBoolean()
```

Comment:
point of documentation.  If you have:

```
streams.sink.enabled=true
streams.sink.enabled.to.foo=false
```

Then sink is enabled on all databases EXCEPT foo.  (Local overrides global)

Reply:
Done